package.json

```diff
@@ -31,6 +31,7 @@
 		"gulp-rename": "^2.0.0",
 		"gulp-sass": "^5.1.0",
 		"gulp-typescript": "^6.0.0-alpha.1",
+		"npm-watch": "^0.11.0",
```

Comment:
it is not necessary, take care of add unnecesary code or dependencies to our project